plugins-ref/premium/powerpaste/powerpaste_commands_events_apis.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: default
+title: Commands, Events and API Methods for the PowerPaste plugin
+title_nav: Commands, Events and API
```

Comment:
There is no APIs here so that shouldn't be in the name... the file name for this also likely needs updating as well.

```suggestion
title: Commands and Events for the PowerPaste plugin
title_nav: Commands and Events
```

- [ ] Fix incorrect page/file name